core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -302,3 +433,7 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
     } 
   }
 }
+
+class TestTimeoutException (msg:String)
+  extends java.lang.RuntimeException (msg) {}
```

Comment:
This should be simply:

```scala
class TestTimeoutException (msg: String) extends java.lang.RuntimeException (msg)
```

It seems like we use this in another test too, right? If so, we should move it to its own file.